quickstarts/microsoft.dataprotection/backup-create-storage-account-enable-protection/main.bicep

```diff
@@ -43,28 +80,179 @@ resource backupVault 'Microsoft.DataProtection/backupVaults@2021-01-01' = {
   }
 }
 
-resource backupPolicy 'Microsoft.DataProtection/backupVaults/backupPolicies@2021-01-01' = {
-  parent: backupVault
-  name: backupPolicyName
+resource vaultName_backupPolicy 'Microsoft.DataProtection/backupVaults/backupPolicies@2022-05-01' = {
+  parent: vault
+  name: '${backupPolicyName}'
```

Comment:
remove string interpolation since the only member of this expression is a single variable

Reply:
Addressed